### ISSUE TITLE
DApp error when creating listing with Metamask 4.12.0

### ISSUE THREAD

**Issue by franckc** [OriginProtocol/origin#673]
When creating a listing on Mainnet with Metamask 4.12.0, I get an error (see screenshot).
Note the that listing does get published but it is a failure related to confirmation callback.

<img width="1488" alt="screen shot 2018-10-03 at 11 30 53 am" src="https://user-images.githubusercontent.com/945910/46431236-20432980-c700-11e8-919f-94e9cbb55a7c.png">

**Comment by franckc** [comment]
CC @wanderingstan @sparrowDom

**Comment by wanderingstan** [comment]
The errors are:
```
MetaMask - RPC Error: Internal JSON-RPC error.
t @ chrome-extension://nkbihfbeogaeaoehlefnkodbefgpgknn/inpage.js:1
```
^ Seems to happen _inside_ metamask.
 
```
Error: Failed to subscribe to new newBlockHeaders to confirm the transaction receipts.
```

**Comment by franckc** [comment]
Changed priority to p1 since Metamask 4.13 was released and it does not have this issue.

**Comment by sparrowDom** [comment]
This pull request solves the issue with MetaMask 4.12.0  listing creation ( https://github.com/OriginProtocol/origin-js/pull/599 ). It suppresses that specific error and implements a workaround. 

We have yet to decide if we want to implement the workaround or just not support the broken version of Metamask.

**Comment by micahalcorn** [comment]
Resolved by #803 ✅
